internal/configs/mock_provider.go

```diff
@@ -393,12 +450,20 @@ func decodeOverrideDataBlock(block *hcl.Block, source OverrideSource) (*Override
 	return override, diags
 }
 
+var (
+	// When this attribute is set to true, the values specified in the override
+	// block will be used for computed attributes even when planning. Otherwise,
+	// the computed values will be set to unknown, just like in a real plan.
+	overrideComputed = "override_computed"
```

Comment:
nit: I like to put the variables and consts at the top of the file instead of just floating in the middle, but I don't think we say that's actually required anywhere.